src/modules/folder/folder.model.ts

```diff
@@ -54,11 +56,16 @@ export class FolderModel extends Model implements FolderAttributes {
   @BelongsTo(() => UserModel)
   user: UserModel;
 
+  @HasOne(() => PrivateSharingFolderModel, {
+    constraints: false,
+  })
+  privateSharingFolder: PrivateSharingFolderModel;
```

Comment:
I think, `hasMany` should be here, as the folder could be shared with multiple users.